### DISCUSSION TITLE
Create a Tasks using the API

### DISCUSSION THREAD

**Discussion by cristianSysadmin** [YukiGasai/obsidian-google-calendar#103] [General]
I need to create a new task in Google Tasks, like this: 
https://github.com/YukiGasai/obsidian-google-calendar/blob/master/docs/Api/Create.md

Do you know if this is possible?

**Comment by YukiGasai** [comment]
This would be a function of the [Google Task](https://github.com/YukiGasai/obsidian-google-tasks) Plugin, but it is not planned. You can implement this function yourself. I guess you are using dataview or templater?

~~~
```dataviewjs
const token = window.localStorage.getItem("googleTaskAccessToken")

const tasklist = "YOUR TASKLIST ID"

const url = `https://tasks.googleapis.com/tasks/v1/lists/${tasklist}/tasks`

const body = {
	"title": "Hello World"
}

const request = fetch(url, {
	method: "POST",
	headers: {
		"Content-Type": "application/json",
	    "Authorization": `Bearer ${token}`
	},
	body: JSON.stringify(body)
})

```
~~~

The option to get the token from localStorage could become deprecated at some point.
